08-generic_cache/01_naive/cache_internal_test.go

```diff
@@ -0,0 +1,45 @@
+package cache
```

Comment:
All caches in 01 are `[int, string]`. We should have some variety as early as there (otherwise, the generics aren't really needed)